libraries/ms-drivers/src/stm32f0xx/spv1020_mppt.c

```diff
@@ -0,0 +1,48 @@
+#include "spv1020_mppt.h"
+#include "spv1020_mppt_defs.h"
+
+// This file implements stm32-specific functionality. See ../spv1020_mppt.c for the rest.
+
+// Send a command to SPV1020 and get a response. |rx_len| must be <= 2.
+// It's safe to pass NULL as |rx_data| if |rx_len| is 0.
+static StatusCode prv_send_command(SpiPort port, uint8_t command, uint8_t *rx_data, size_t rx_len) {
+  if (port >= NUM_SPI_PORTS) {
+    return status_code(STATUS_CODE_INVALID_ARGS);
+  }
+
+  // We don't use spi_exchange since that assumes 0x00 is NOP, whereas SPV1020 appears to use 0x01.
```

Comment:
I'm not sure this is correct. In the command list NOP is 0x01, but on page 12 paragraph 3 it says:
> Therefore the master appends a number of NOPs (0x00) to the command so that the entire data can be transmitted

It makes a lot more sense to me for this to be left as 0x00, matching spi_exchange. I think we should request a `smoke_spi` test with exchange before jumping into this weird custom spi implementation.

My theory is that the *command word* NOP is different from the SPI NOP, and that the SPI NOP is really 0x00.

I hate this datasheet.